archinstall/lib/user_interaction/general_conf.py

```diff
@@ -172,8 +172,8 @@ def ask_additional_packages_to_install(pre_set_packages: List[str] = []) -> List
 
 	def read_packages(already_defined: list = []) -> list:
 		display = ' '.join(already_defined)
-		input_packages = TextInput(_('Write additional packages to install (space separated, leave blank to skip): '), display).run()
-		return input_packages.split(' ') if input_packages else []
+		input_packages = TextInput(_('Write additional packages to install (space separated, leave blank to skip): '), display).run().strip()
```

Comment:
You probably want `.replace("  ", " ").strip()`

Reply:
I think .split() without ' ' as an argument treats multiple spaces as one.